tests/e2e/systemtest/test_noobaa_system.py

```diff
@@ -0,0 +1,39 @@
+import logging
+
+from ocs_ci.framework.testlib import E2ETest
+from ocs_ci.ocs.bucket_utils import (
+    write_data_to_buckets_and_check_integrity,
+    verify_s3_buckets_integrity,
+)
+
+log = logging.getLogger(__name__)
+
+
+class TestNoobaaSystem(E2ETest):
+    """
+    Test Noobaa System
+
+    """
+
+    def test_noobaa_system(
+        self,
+        bucket_factory,
+        mcg_obj,
```

Comment:
`bucketclass_dict` is not passed as a parameter. The test needs to be parametrized and the signature needs to change in order to support it.